app/components/ActiveIndicator/ActiveIndicator.tsx

```diff
@@ -9,10 +9,10 @@ const ActiveIndicator = ({ isActive, className }: Props) => {
   return (
     <div
       className={cn(
-        "w-2.5 h-2.5 rounded-full border-[1px] border-[#333333] absolute top-0 left-0 z-[1]",
+        "w-2.5 h-2.5 rounded-full border-[1px] border-grey-800 absolute top-0 left-0 z-[1]",
```

Comment:
```suggestion
        "w-2.5 h-2.5 rounded-full border-[1px] border-gray-800 absolute top-0 left-0 z-[1]",
```

for all `grey` -> `gray`; hopefully relatively easy change with grep